released/packages/coq-hanoi/coq-hanoi.1.0.0/opam

```diff
@@ -0,0 +1,41 @@
+# This file was generated from `meta.yml`, please do not edit manually.
+# Follow the instructions on https://github.com/coq-community/templates to regenerate.
+
+opam-version: "2.0"
+maintainer: "thery@sophia.inria.fr"
+version: "dev"
```

Comment:
```suggestion
opam-version: "2.0"
maintainer: "thery@sophia.inria.fr"
```
Unfortunately we don't yet have the automation to remove unwanted boilerplate from opam release packages.

Reply:
should be ok now